deltachat-rpc-client/tests/test_folders.py

```diff
@@ -143,7 +143,10 @@ def test_delete_deltachat_folder(acfactory, direct_imap):
     # Wait until new folder is created and UIDVALIDITY is updated.
     while True:
         event = ac1.wait_for_event()
-        if event.kind == EventType.INFO and "uid/validity change folder DeltaChat" in event.msg:
+        if (
+            event.kind == EventType.INFO
+            and "UID validity for folder DeltaChat and transport 1 changed from " in event.msg
```

Comment:
Nitpick: "transport 1: UID validity for folder DeltaChat changed from " may be better because the folder is inside the transport